kmmresult/src/commonMain/kotlin/at/asitplus/KmmResult.kt

```diff
@@ -268,7 +268,7 @@ inline fun <R, T : R> KmmResult<T>.recoverCatching(block: (error: Throwable) ->
 /**
  * Non-fatal-only-catching version of stdlib's [runCatching], directly returning a [KmmResult] --
  * Re-throws any fatal exceptions, such as `OutOfMemoryError`. Relies on [Arrow](https://arrow-kt.io)'s
- * [nonFatalOrThrow](https://apidocs.arrow-kt.io/arrow-core/arrow.core/non-fatal-or-throw.html) internally.
+ * [nonFatalOrThrow](https://apidocs.arrow-kt.io/arrow-core/arrow.core/non-fatal-or-throw.html) logic.
```

Comment:
This doesn't make it clear that Arrow is not a dependency imo, and where we took the list from doesn't really matter _enough_ that we'd put it in the docs.

How about `A list of fatal exceptions is [here](https://github.com/a-sit-plus/KmmResult/tree/main/bla/blub).`?

Reply:
I want to give credit, where credit is due. how about "Mirrors Arrow's logic"

Reply:
if you link to our source for `nonFatalOrThrow`, the comment there references arrow; is that not enough?

Reply:
I'm gonna go with re-iplements Arrow's logic to avoid a dependency on Arrow for a single function.

Reply:
sounds good to me